config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root "home#index"
```

Comment:
By specifying the root route this way you are saying for rails to look for an index method in the Home Controller. This is leading to an error since you do not have such controller.